android/src/test/java/dev/openfeature/sdk/DeveloperExperienceTests.kt

```diff
@@ -258,11 +258,9 @@ class DeveloperExperienceTests {
         waitAssert {
             assertEquals(OpenFeatureStatus.NotReady, OpenFeatureAPI.getStatus())
         }
-        testScheduler.advanceTimeBy(1)
         waitAssert {
-            assertEquals(OpenFeatureStatus.NotReady, OpenFeatureAPI.getStatus())
+            assertTrue(OpenFeatureAPI.getStatus() is OpenFeatureStatus.Error)
```

Comment:
I'm having trouble understanding this change. Could you please quickly explain why it was made?